src/test/scala/NomisSpec.scala

```diff
@@ -36,7 +36,7 @@ class NomisSpec extends FunSpec with BeforeAndAfterAll with GivenWhenThen with M
       val minuteAgo = rightNow.minus(6000)
 
       When("Case Notes are pulled from the API")
-      val result = Await.result(source.pull(minuteAgo, rightNow), 5.seconds)
+      val result = Await.result(source.pull(minuteAgo, rightNow), 10.seconds)
```

Comment:
Nothing to do with this change - CircleCI is cpu constrained to these tests sometimes takes longer than 5 seconds